spec/Domain_Maps.tex

```diff
@@ -144,7 +144,7 @@ \section{Domain Maps for Domain Types}
 
 The following syntactic sugar is provided within the \chpl{dmapped} clause.
 If a \chpl{dmapped} clause starts with the name of a domain map class,
-it is considered to be a constructor expression as if preceded by
+it is considered to be an initializer expression as if preceded by
```

Comment:
Should this be "an initializer *call*" instead of "an initializer *expression*"?  That phrasing just feels weird to me

Reply:
"initialization expression" ?

Reply:
Works for me!